### PR TITLE
Update README.rst using OCA format

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: currency_rate_update/README.rst [hunk 1]
```diff
@@ -40,24 +44,57 @@ You can active or deactivate the update.
...
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+>>>>>>> 051785f... Update README.rst using OCA format:currency_rate_update/README.rst
```
Comment (adrienpeiffer) [inline currency_rate_update/README.rst]: Can you remove this line ?

Comment (yvaucher) [inline currency_rate_update/README.rst]: Thanks I amended my commit

**Comment by adrienpeiffer** [comment]
Thanks ! :+1:

File: currency_rate_update/README.rst [hunk 2]
```diff
@@ -40,24 +44,56 @@ You can active or deactivate the update.
...
+
 The module supports multi-company currency in two ways:
 
-*    the currencies are shared, you can set currency update only on one
-     company
-*    the currency are separated, you can set currency on every company
-     separately
+* the currencies are shared, you can set currency update only on one
+  company
+* the currency are separated, you can set currency on every company
```
Comment (pedrobaeza) [inline currency_rate_update/README.rst]: s/currency/currencies

**Comment by yvaucher** [comment]
Fixed that plural and replaced OpenERP by Odoo

**Comment by yvaucher** [comment]
Just unsure about "on the regular daily concertation" that I corrected to "on the regular daily consultation"

Maybe "daily check" would be better ?

**Comment by pedrobaeza** [comment]
Yes, better "daily check" or "daily query"

**Comment by yvaucher** [comment]
@pedrobaeza fixed

**Comment by pedrobaeza** [comment]
:+1: 

I fast-track, as Travis result doesn't change with these files

Next time, please use `git mv` command to avoid the file deletion/addition.

**Comment by yvaucher** [comment]
@pedrobaeza I used `git mv`, just didn't wanted to make 2 separate commit for the readme file

**Comment by pedrobaeza** [comment]
Ahm, curious that git doesn't detect then that it's the same file. BTW, it doesn't matter

**Pull request merged**